services.rst

```diff
@@ -64,48 +64,67 @@ server in the NTP configuration page.
-   4. ads. This mode instructs Samba to act as a domain member in an Active Directory Server (ADS) realm.
+Next, as correct time synchronization with the AD domain is necessary for good
+performance, the NTP service should be configured and turned ON. Ideally, both
+the AD domain and Rockstor machines should thus use the same NTP time server.
 
-* **Realm**: When the ads Security Model is selected, this allows you to
-  specify the ADS Realm the Samba server should act as a domain member of.
-* **Template shell**: The login shell for the user
-* **Allow offline login**
+Now that the Samba workgroup has been defined, and the NTP service configured
+and running, the AD service can be configured. To do so, click on the
+**wrench** icon and fill the form with the values corresponding to your AD
+domain.
 
-To start or stop the service, click the corresponding ON or OFF button.
-
-For example, for an environment with the Winbind domain as *rockstorad*, and the AD controller as *WIN-H323VCUT6GT.ROCKSTORAD.LOCAL*, these are the values used to configure AD.
+.. image:: images/services/ad_config.png
+   :scale: 70 %
+   :align: center
 
-    * Winbind Domain - rockstorad
-    * Security Model - ads
-    * Winbind ADS Realm - rockstorad.local
-    * Winbind Domain Controllers - WIN-H323VCUT6GT.ROCKSTORAD.LOCAL
-    * Template shell - /bin/bash
-    * Don't check Allow offline login box.
+The individual fields of the form are described below.
 
-Once the AD service is configured, click *Join Domain* and enter the AD Administrator username and password, and click Submit. The appliance will now attempt to join the domain and will display a *Join Ok* message if successful.
+* **Domain/Realm name**: Specifies the Windows Active Directory or domain
+  controller to which to connect.
+* **Administrator Username**:  Name of the user to use for the enrollment to
+  the AD. Tihs should be the AD's administrator account.
+* **Allow offline login**: Password for the Administrator username.
+* **Enable enumeration**: Fetch and display all users/groups values. As this
+  option can have a notable performance cost in some servers (with high number
+  of users, for instance), this option is disabled by default. Note, however,
+  that this option must be enabled for Rockstor to be able to list AD users and
+  groups in the web-UI. See `SSSD FAQ <https://sssd.io/docs/users/faq.html#when-should-i-enable-enumeration-in-sssd-or-why-is-enumeration-disabled-by-default>`_ for
+  further details.
+* **Disable automatic ID mapping**: By default, the AD provider will map UID
+  and GID values from the objectSID parameter in Active Directory. Check this
+  option if you want to disable ID mapping and instead rely on POSIX attributes
+  defined in Active Directory. See `SSSD documentation <https://linux.die.net/man/5/sssd-ad>`_ for
+  furhter details.
+* **Treat user and group names as case-sensitive**
+
+Rockstor relies on `SSSD <https://sssd.io/>`_ for the management of identities
```

Comment:
```suggestion
Rockstor 4 relies on `SSSD <https://sssd.io/>`_ for the management of identities
```
Just in case our legacy users see this and end up going down the wrong path. The rest pertains to obviously different stuff but given this is a directive pertaining to the cli, there is no telling.